global name 'gcode' is not defined

Using pronterfyce.py for printing. After each finished print, following error appears in console (print time used only as reference):

```
...
Print ended at: 20:26:10 and took 0:03:16
[ERROR] Print end callback failed with:
Traceback (most recent call last):
  File "/home/diggit/Downloads/Printrun/printrun/printcore.py", line 543, in _print
    try: self.endcb()
  File "/home/diggit/Downloads/Printrun/printrun/pronterface.py", line 1602, in endcb
    pronsole.pronsole.endcb(self)
  File "/home/diggit/Downloads/Printrun/printrun/pronsole.py", line 1209, in endcb
    if(len(gcode.filament_length_multi)>1):
NameError: global name 'gcode' is not defined
```

I have no spools defined.

Introduced in 5db1366c0a5cf107ec6218c82847953c78f0dcda
Can you take a look @rockstorm101 ?

--- a/printrun/pronsole.py
+++ b/printrun/pronsole.py
@@ -1206,8 +1206,8 @@ class pronsole(cmd.Cmd):
 
                 # Update the length of filament in the spools
                 self.spool_manager.refresh()
-                if(len(gcode.filament_length_multi)>1):
-                    for i in enumerate(gcode.filament_length_multi):
+                if(len(self.fgcode.filament_length_multi)>1):
+                    for i in enumerate(self.fgcode.filament_length_multi):
                         if self.spool_manager.getSpoolName(i[0]) != None:
                             self.spool_manager.editLength(
                                 -i[1], extruder = i[0])

